src/js/bind.js

```diff
@@ -26,14 +26,14 @@ export function bindAdvance() {
   const handler = _setupAdvanceOnHandler.call(this, selector);
 
   // TODO: this should also bind/unbind on show/hide
-  if (!isUndefined(selector)) {
-    const el = document.querySelector(selector);
+  const el = document.querySelector(selector);
+  if (!isUndefined(selector) && el) {
```

Comment:
event handlers lost on any selector that is null at the time, since the step might not be in the DOM yet.